suite-native/transactions/src/components/TransactionsList/TransactionListItemContainer.tsx

```diff
@@ -166,9 +166,13 @@ export const TransactionListItemContainer = ({
     );
 
     const iconColor: Color = isTransactionPending ? 'backgroundAlertYellowBold' : 'iconSubdued';
-    const coinSymbol = tokenTransfer?.contract ?? networkSymbol;
+    const coinSymbol = !isPhishingTransaction
```

Comment:
nit: Not negated condtionals are better readable. Please change it to `isPhishingTransaction ? B : A`

Reply:
4a845d4e24